stylelint.config.js

```diff
@@ -9,25 +9,21 @@ declaration-block-no-redundant-longhand-properties -
 declaration-empty-line-before -
   Turned off.
   TODO: Turn on this rule and work out what style we want.
-declaration-property-value-no-unknown -
-  Turned off for Less per documentation guidance.
 media-feature-range-notation -
-  Prefer prefixed values, since Less doesn't support ranges.
-media-query-no-invalid -
-  Turned off because of https://github.com/ssivanatarajan/stylelint-less/issues/6
+  Prefer prefixed values, traditionally used since Less doesn't support ranges.
```

Comment:
Possibly revisit this?

Reply:
Yea revisiting several of those options would be a good idea. Some of them may generate a lot that needs updating if they are changed.